style.css

```diff
@@ -0,0 +1,767 @@
+:root {
+  --gradient: linear-gradient(-45deg, #131313, #e1d9dc, #000000, #dce6e4);
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  text-decoration: none;
+}
+
+body {
+  padding: 0;
+  margin: 0;
+}
+
+.material-symbols-outlined {
+  font-variation-settings: "FILL" 0, "wght" 400, "GRAD" 0, "opsz" 24;
+}
+
+.navigation-bar {
+  z-index: 2;
+  top: 0;
+  position: sticky;
+  max-height: 5rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: var(--gradient);
+  background-size: 400% 400%;
+  animation: color-changer 10s ease infinite;
```

Comment:
Mi se pare neprofesionistă această animație. Poate s-ar simți mai bine dacă ar fi mai înceată și ai folosi culori mai puțin contrastante în gradient.